### DISCUSSION TITLE
Integrate Filecoin as a decentralised layer to store users' data in a company

### DISCUSSION THREAD

**Discussion by danielezzzz** [filecoin-project/community#577] [General]
Hello. Let's say I have a company that builds a mobile application. The user is required to sign in to his account on our website, via desktop, and upload a file. The file has to be visible in the user account on the mobile application. In order to preserve privacy and security, I would like to use a decentralized cloud to store users' files. Is this something that can be done using Filecoin?

**Comment by IdaParks** [comment]
Filecoin is a decentralized storage network that allows users to store and retrieve data in a secure and distributed manner. It utilizes blockchain technology and a unique incentive mechanism to create a decentralized marketplace for storage providers and users in [mcdvoice](https://mcdvoice.me/).
